tests/test_lhs.py

```diff
@@ -0,0 +1,67 @@
+import unittest
+from pyDOE3.doe_lhs import lhs
+from pyDOE3.doe_lhs import _lhsclassic
+from pyDOE3.doe_lhs import _lhscentered
+from pyDOE3.doe_lhs import _lhsmaximin
+import numpy as np
+
+class TestLhs(unittest.TestCase):
+
+    #
+    #function to compare two arrays
+    #
+    def arrayEqual (self, actual, expected):
+        np.testing.assert_allclose(actual, expected)
```

Comment:
I would prefer to inlline the `np.testing.assert_allclose` and remove this function